js/checklistcongres.js

```diff
@@ -0,0 +1,40 @@
+//#region ChecklistCongres
+const checkboxes = document.querySelectorAll('.checkcongres');
+const helpicons = document.querySelectorAll('.help-icon');
+
+if(localStorage.length > 0){
+    const keys = Object.keys(localStorage);
+
+    keys.forEach(key => {
+        const value = localStorage.getItem(key);
+        document.getElementById(key).checked = value;
```

Comment:
pas de null check ? j'ai une clé pour le darkmode, l'id correspond à rien en html normalement, ça claque pas une erreur console ?

Reply:
![image](https://user-images.githubusercontent.com/35492348/230570324-54422675-7496-43f4-9ee3-841a1c118deb.png)
i fixed 2 bugs, modal loading  due to code isolation in new file, and what i tell you previously